colp_build/sql/name.sql

```diff
@@ -0,0 +1,110 @@
+		                                                                                    'AT'
+		                                                                                    ),
+		                                                                                'HSES',
+		                                                                                'HOUSES'
+		                                                                                ),
+		                                                                            '\s\(GROUP 1\s',
+		                                                                            '\s(GROUP 1)\s'
+		                                                                            ),
+		                                                                        '\sPK\s|\sPK-',
+		                                                                        ' PARK '
+		                                                                        ),
+		                                                                    'STA\s',
+		                                                                    'STATION '
+		                                                                    ),
+		                                                                'LIBR\s',
+		                                                                'LIBRARY '
+		                                                                ),
+		                                                            'INDUST\sPARK|IND\.\sPARK',
+		                                                            'INSUSTRIAL PARK'
+		                                                            ),
+		                                                        'PCT\s|PCT\.\s',
+		                                                        'PRECINCT '
+		                                                        ),
+		                                                    'N\.Y\.C\.P\.D\.|N\.Y\.P\.D\.|NYCPD',
+		                                                    'NYPD'
+		                                                    ),
+		                                                'F\.H\.A\.\.',
+		                                                'FHA '
+		                                                ),
+		                                            'GDN\s|GARD\s',
+		                                            'GARDEN '
+		                                            ),
+	                                            'AVE\)|AVE\.\)',
+	                                            'AVENUE)'
+	                                            ),
+                                            '\(AVE|\(AVE\.',
+                                            '(AVENUE'
+                                            ),
+                                        '\sAVE\s|\sAVE\.\s|\sAVE-\s|\sAVE\.\s|\sAV\s',
+                                        ' AVENUE '
```

Comment:
60 BROAD ST/33 BEAVER ST is still becoming 60 BROAD ST/33 BEAVENUE ST
MOTT HAVEN BRANCH EXTENSION is still MOTT HAVENUE BRANCH EXTENSION